ApplicationLibCode/ProjectDataModel/Summary/RimEnsembleCurveFilter.cpp

```diff
@@ -610,7 +629,7 @@ std::vector<RimSummaryCase*> RimEnsembleCurveFilter::applyFilter( const std::vec
             }
             else if ( eParam.isNumeric() )
             {
-                if ( !crpValue.isNumeric() || crpValue.numericValue() < m_minValue() || crpValue.numericValue() > m_maxValue() )
+                if ( !crpValue.isNumeric() || crpValue.numericValue() < minValue() || crpValue.numericValue() > maxValue() )
```

Comment:
Should we have a helper function "inSelectedRange()" or something that could be used with the value range slider field to avoid repeating these kinds of checks?

i.e. 
```
if (m_valueRangeSlider.isInSelectedRange(value))
{
...
}
```

Reply:
Good suggestion, added a helper method.